docs/network-flow-visibility.md

```diff
@@ -754,6 +755,31 @@ Mouse out or click on the background will bring all the traffic back.
 
 <img src="https://downloads.antrea.io/static/08032022/flow-visibility-np-3.png" width="900" alt="Network-Policy Flows Dashboard">
 
+#### Network Topology Dashboard
+
+Network Topology Dashboard visualizes Pod-to-Pod and Pod-to-Service traffic,
+including intra-Node traffic as well. Pods labels are visually grouped by Node.
+Each arrow points from the source to the destination, and is labelled by the
+amount of data transmitted.
+
+<img src="" width="900" alt="Network Topology Dashboard">
+
+Users can filter graph results based on the number of flow records, as well as
```

Comment:
```suggestion
Users can filter graph results based on the number of flow records.
```
not just source and destination Pod names but all the fields? we can just remove it, as the same filter has been introduced in the above dashboard.